grease/src/Grease/FunctionOverride.hs

```diff
@@ -371,7 +371,7 @@ callAssert bak mvar mmConf archCtx _pfn _pfile _pline ptxt = do
   let sym = C.backendGetSym bak
   st0 <- get
   let endian = Symbolic.fromCrucibleEndian (archCtx ^. archEndianness)
-  (txt, st1) <- liftIO $ loadConcreteString bak mvar mmConf endian ptxt st0
+  (txt, st1) <- liftIO $ loadConcreteString bak mvar mmConf endian ptxt Nothing st0
```

Comment:
No need to change this necessarily, but I sometimes like giving things names to make it clear what they represent
```suggestion
  let maxSize = Nothing
  (txt, st1) <- liftIO $ loadConcreteString bak mvar mmConf endian ptxt maxSize st0
```
An alternative would be to make a new `data` type, but that feels perhaps excessive in this case.